docs/issue-triage.md

```diff
@@ -0,0 +1,10 @@
+# Issue Triage
+
+On a regular basis (target: weekly) the development team will triage issues matching the following [GitHub search](https://github.com/Azure/azure-service-operator/issues?q=is%3Aissue+is%3Aopen+-label%3Atriaged+no%3Amilestone+):
```

Comment:
Minor, you could rephrase this as:
```suggestion
On a regular basis (target: weekly) the development team will triage issues matching the following search criteria [is:issue is:open -label:triaged no:milestone](https://github.com/Azure/azure-service-operator/issues?q=is%3Aissue+is%3Aopen+-label%3Atriaged+no%3Amilestone+)
```

and do away entirely with the ``` section below

Reply:
I think this looks a bit cleaner in that it turns the issue criteria into the link rather than having a separate "this search" link

Reply:
I originally wrote it the way you suggest - but broke it out the way that I did so that the search criteria was more explicitly called out for people to see. Makes it easier to copy too.